NEWS.md

```diff
@@ -1,3 +1,28 @@
+Unreleased
+
+* Remove `suspenders` system executable
+* Introduce `suspenders:accessibility` generator
+* Introduce `suspenders:inline_svg` generator
+* Introduce `suspenders:factories` generator
+* Introduce `suspenders:advisories` generator
+* Introduce `suspenders:styles` generator
+* Introduce `suspenders:jobs` generator
+* Introduce `suspenders:lint` generator
+* Introduce `suspenders:rake` generator
+* Introduce `suspenders:views` generator
+* Introduce `suspenders:setup` generator
+* Introduce `suspenders:tasks` generator
+* Introduce `suspenders:db:migrate` task
+* Introduce `suspenders:email` generator
+* Introduce `suspenders:testing` generator
+* Introduce `suspenders:prerequisites` generator
+* Introduce `suspenders:ci` generator
+* Introduce `suspenders:cleanup:organize_gemfile` task
+* Introduce `suspenders:environments:production` generator
+* Introduce `suspenders:environments:test` generator
+* Introduce `suspenders:environments:development` generator
+* Introduce `suspenders:install:web` generator
```

Comment:
I should add that we added an [application template](https://guides.rubyonrails.org/rails_application_templates.html) too.

Reply:
a38f247e9a4e0525d35dc28bd55618c279cba5f7